### ISSUE TITLE
No such file model/coco_valid1_eval.pkl

### ISSUE THREAD

**Issue by SilencerChen** [gujiuxiang/MIL.pytorch#7]
I try to run the test code but couldn't find model/coco_valid1_eval.pkl file.  Could you updata it ? Thanks so much.

**Comment by YuanEZhou** [comment]
I have the same issue. @SilencerChen  @gujiuxiang

**Comment by gujiuxiang** [comment]
@SilencerChen @YuanEZhou, uploaded, pls find the file in Weiyun. Thanks.

**Comment by winnechan** [comment]
i have create a pkl file that can be loaded without coco_voc by only storing the .details.precision details.score numpy.array.

https://drive.google.com/open?id=1eRrH45xOj41__v2bXMVa8OGZ1Gj6NYze

using python 3 to load the python 2 pkl file:

`with open(pickle_file_name, 'rb') as f:`
&emsp;&emsp;`d = cPickle.load(f, encoding='latin1')`

but there's a problem i don't know why: i can get reasonable results by removing the softmax layer in the model. 

another problem is that, to me, the output before softmax seems to be the noisy-OR pooling of each image region instance, which can be used to be the final output directly without the score matching of unknown meaning.

can someone explain?
